src/pudl/transform/vcerare.py

```diff
@@ -0,0 +1,330 @@
+"""Transformations of the Vibrant Clean Energy renewable generation profiles.
+
+Wind and solar profiles are extracted separately, but concatenated into a single table
+in this module, as they have exactly the same structure.
+"""
+
+import pandas as pd
+from dagster import AssetCheckResult, asset, asset_check
+
+import pudl
+from pudl.helpers import cleanstrings_snake, simplify_columns, zero_pad_numeric_string
+from pudl.metadata.dfs import POLITICAL_SUBDIVISIONS
+
+logger = pudl.logging_helpers.get_logger(__name__)
+
+
+def _prep_lat_long_fips_df(raw_vcegen__lat_lon_fips: pd.DataFrame) -> pd.DataFrame:
+    """Prep the lat_long_fips table to merge into the capacity factor tables.
+
+    Prep entails making sure the formatting and column names match those in the
+    capacity factor tables, adding 0s to the beginning of FIPS codes with 4 values,
+    and making separate county/subregion and state columns.
+
+    The county portion of the county_state column does not map directly to FIPS ID.
+    Some of the county names are actually subregions like cities or lakes. For this
+    reason we've named the column county_or_lake_name and it should be considered
+    part of the primary key. There are several instances of multiple subregions that
+    map to a single county_id_fips value.
+    """
+    logger.info(
+        "Preping Lat-Long-FIPS table for merging with the capacity factor tables"
+    )
+    ps_usa_df = POLITICAL_SUBDIVISIONS[POLITICAL_SUBDIVISIONS["country_code"] == "USA"]
+    state_names = cleanstrings_snake(
+        ps_usa_df, ["subdivision_name"]
+    ).subdivision_name.tolist()
+    state_pattern = "|".join(state_names)
+    # Transform lat long fips table by making the county_state_names lowercase
```

Comment:
Non-blocking: It'd be more helpful to split these docstrings and add each on the line where the change is being made, since you're chaining many steps here.